internal/services/notification/db/objects/objects.go

```diff
@@ -35,4 +35,10 @@ var ObjectsInfo = []sqlg.ObjectInfo{
 			"create index if not exists runwebhookdelivery_sequence_idx on runwebhookdelivery(sequence)",
 		},
 	},
+	{
+		Name: "LastRunEvent", Table: "lastrunevent",
```

Comment:
This is a "strange" table since it'll contain at most one row. It'll be good to document this. As an additional note, in the documentation, we could add that if in future we'll need to store multiple last sequences like this we could reuse it adding a specific key.